docs/oscar_references.bib

```diff
@@ -2290,6 +2290,18 @@ @Article{SV-D-V87
   zbmath        = {4069055}
 }
 
+@Article{SW05,
+  author        = {Speyer, David and Williams, Lauren},
+  title         = {The {{Tropical Totally Positive Grassmannian}}},
```

Comment:
```suggestion
  title         = {The Tropical Totally Positive Grassmannian},
```
our bibtex parser does not work well with braces inside of fields. It will keep the casing even without them (contrary to latex)